### PR TITLE
Add glog/flag workaround to controller-manager

### PR DESCRIPTION
**Pull request by justinsb**
Per https://github.com/kubernetes/kubernetes/issues/17162

```release-note
NONE
```

### CODE REVIEW

**Comment by roberthbailey** [comment]
Do we not need this in https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/apiserver/main.go or https://github.com/kubernetes-sigs/cluster-api/blob/master/clusterctl/main.go too?

**Comment by justinsb** [comment]
We likely do @roberthbailey - I just hadn't gotten that far yet... I'll add it!

File: cmd/apiserver/main.go [hunk 1]
```diff
@@ -14,13 +14,14 @@ See the License for the specific language governing permissions and
...
-
 package main
 
 import (
+	"flag"
+
 	// Make sure glide gets these dependencies
-	_ "k8s.io/apimachinery/pkg/apis/meta/v1"
 	_ "github.com/go-openapi/loads"
+	_ "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (roberthbailey) [inline cmd/apiserver/main.go]: this import order looks funny -- github then k8s.io then github then k8s.io. Can you group the two github ones and the two k8s.io ones in two blocks?

**Comment by justinsb** [comment]
clusterctl has it already: https://github.com/kubernetes-sigs/cluster-api/blob/master/clusterctl/cmd/root.go#L52

But I added it to apiserver.

File: cmd/apiserver/main.go (hunk 1 above)
Comment (justinsb) [reply cmd/apiserver/main.go]: Sure - done!

**Pull request merged**